Source/ProjectNL/Ai/BTTask/BTTask_DirectTo.cpp

```diff
@@ -0,0 +1,56 @@
+// Fill out your copyright notice in the Description page of Project Settings.
+
+
+#include "ProjectNL/Ai/BTTask/BTTask_DirectTo.h"
+
+#include "AIController.h"
+#include "BehaviorTree/BlackboardComponent.h"
+#include "Kismet/KismetMathLibrary.h"
+
+UBTTask_DirectTo::UBTTask_DirectTo()
+{
+}
+
+EBTNodeResult::Type UBTTask_DirectTo::ExecuteTask(UBehaviorTreeComponent& OwnerComp, uint8* NodeMemory)
+{
+	Super::ExecuteTask(OwnerComp, NodeMemory);
+
+	// Get the AI Controller
+	AAIController* AIController = OwnerComp.GetAIOwner();
+	if (!AIController)
+	{
+		return EBTNodeResult::Failed;
+	}
+
+	// Get the controlled pawn
+	APawn* ControlledPawn = AIController->GetPawn();
+	if (!ControlledPawn)
+	{
+		return EBTNodeResult::Failed;
+	}
+
+	// Get the target actor from the blackboard
+	UBlackboardComponent* BlackboardComp = OwnerComp.GetBlackboardComponent();
+	if (!BlackboardComp)
+	{
+		return EBTNodeResult::Failed;
+	}
+
+	UObject* TargetObject = BlackboardComp->GetValueAsObject("Target"); // Replace "TargetActor" with your actual blackboard key
+	AActor* TargetActor = Cast<AActor>(TargetObject);
+	if (!TargetActor)
+	{
+		return EBTNodeResult::Failed;
+	}
+
+	// Calculate the direction to the target
+	FVector TargetLocation = TargetActor->GetActorLocation();
+	FVector PawnLocation = ControlledPawn->GetActorLocation();
+
+	FRotator LookAtRotation = UKismetMathLibrary::FindLookAtRotation(PawnLocation, TargetLocation);
```

Comment:
```suggestion
	FLocateHelper::GetTargetingRotate(TargetActor, ControlledPawn);
```
동일한 기능을 가진 Helper를 만들어놨으니 이거 써도 될거같아요